src/cli/backup/backup.go

```diff
@@ -236,7 +236,7 @@ func runBackups(
 			Infof(ctx, "Done\n")
 			printBackupStats(ctx, r, string(bo.Results.BackupID))
 		} else {
-			Infof(ctx, "Done - ID: %v\n", bo.Results.BackupID)
```

Comment:
I'm guessing this was to fix the issue with not being able to find user's name in output. I think the problem is probably from us changing where the output is written to. Don't think we should be changing he output format.
/cc @ryanfkeepers in case you have any thoughts

Reply:
Conceptually I'm okay with the change; but I don't recall exactly what it looks like.  For full assurance, I'd like to see a before/after comparison of the different outputs.  That'd be the best way to ensure we're happy with the result.

also, if we keep this change, the duplicate infof should be moved out of the conditional statement.